src/components/Blocking/ReportPostAnonymous.js

```diff
@@ -136,7 +137,7 @@ const styles = StyleSheet.create({
   btnSkipText: {
     fontFamily: fonts.inter[700],
     fontSize: 14,
-    color: '#000',
+    color: COLORS.black,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Great job replacing the hardcoded color with a value from the theme. This improves code readability and maintainability.
```diff
- color: '#000',
+ color: COLORS.black,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->